### DISCUSSION TITLE
Have to click "reactivate" after browser start

### DISCUSSION THREAD

**Discussion by A13xund3r** [dhowe/AdNauseam#2485] [Q&A]
System is Win 7, browser Firefox 115.5.0esr. Can't say when exactly it started, a month or 2 ago. It just stopped filtering then I realized there are 3 reactivate buttons in settings. Clicked them and after browser restart they are here again. WTF? Can't find any info about that.
![reactivate](https://github.com/dhowe/AdNauseam/assets/47088783/580db058-8181-413a-b8e1-31be1fdf3a25)

**Comment by mneunomne** [comment]
Hey @A13xund3r, sorry that you have experienced this. Maybe the adnauseam folder got corrupted somehow, and Firefox is failing to save the information from the addon there. You have some options:
- try to reinstall adnauseam
- clean the extension data adnauseam folder. Here you can find how to locate that folder and try to figure out which of them is adnauseam: https://stackoverflow.com/questions/50706030/where-to-find-location-of-firefox-extension-data-other-than-profilename-browser
- easiest, but maybe most annoying solution if you have many things configured: start from a clean firefox profile.
